CalibTracker/SiStripESProducers/test/mergeBadChannel_cfg.py

```diff
@@ -0,0 +1,53 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("BadChannelMerge")
+process.MessageLogger = cms.Service("MessageLogger",
+    cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO')
+    ),
+    destinations = cms.untracked.vstring('cout')
+)
+process.load('Configuration.Geometry.GeometryIdeal_cff')
+process.load("Configuration.Geometry.GeometryReco_cff")
+
```

Comment:
@suchandradutta in the spirit of cleaning configuration from legacy modules, could you switch here to `process.load("Configuration.Geometry.GeometryRecoDB_cff")` ?